apps/rush-lib/src/cli/scriptActions/BulkScriptAction.ts

```diff
@@ -329,6 +392,13 @@ export class BulkScriptAction extends BaseScriptAction {
         ' For details, refer to the website article "Selecting subsets of projects".'
     });
 
+    this._watchParameter = this.defineFlagParameter({
+      parameterLongName: '--watch',
+      parameterShortName: '-w',
+      description:
+        'Activates a file system watcher to dynamically re-invoke the command when projects change.'
```

Comment:
Consider adding more detail/making it more clear as to what it is that --watch is doing? 
Watches for file changes and dynamically re-invokes the command that --watch is used with. 
Don't like this that much either. Maybe we only describe it in terms of rush build to provide clarity?

Reply:
Updated a bit. We will need to add a companion article.